fab/pillows/production.yml

```diff
@@ -1,19 +1,10 @@
 # fab/pillows/production.yml
 ---
-exclude_groups:
-  - mvp_indicators
```

Comment:
Looks like @krsdimagi said we're only supporting their reports up until 2/28 so not going to add this pillow back in. Going to leave deleting mvp reporting code for someone who will find it more cathartic